### DISCUSSION TITLE
We have to plan the hosting platform to use in 'production' 👷‍♂️🏗️

### DISCUSSION THREAD

**Discussion by Im-Boss-13** [DawnBee/college-medicine-app#118] [Q&A] [answered]
Any suggestions regarding this @DawnBee ?

**Comment by DawnBee** [comment, marked as answer]
> Any suggestions regarding this @DawnBee ?

Just this very day @Im-Boss-13 I have researched some potential hosting platforms that we can use with the project. I narrowed it down to:

- Heroku
- Render
- Vercel (FrontEnd) Heroku (BackEnd)

**Reply by Im-Boss-13** [reply]
Good to hear that! 💪😎

**Reply by DawnBee** [reply]
We have to dive deeper on this topic though, in a not so distant future! 🤣😆

**Reply by Im-Boss-13** [reply]
We're leaving this open for now then!
